src/main/java/services/GetConnection.java

```diff
@@ -20,6 +23,11 @@
       return value;
   }
 
+  /**
+   * getConnection using default username and password
```

Comment:
I think this comment might be unnecessary. The method name kinda explains itself.